examples/sdk/train_api.ipynb

```diff
@@ -12,12 +12,13 @@
   },
```

Comment:
<p>Could you preapre dedicated calls for <code>s3</code> and <code>huggingface</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>